config/migrations/2024/20241212193803-delete-agb-stekene.sparql

```diff
@@ -0,0 +1,124 @@
+# Remove forward properties + mock person/account
+DELETE {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?org ?orgP ?orgO .
+  }
+
+  GRAPH ?g {
+    ?mockPerson ?mockPersonP ?mockPersonO .
+    ?mockAccount ?mockAccountP ?mockAccountO .
+  }
+}
+WHERE {
+  BIND(<http://data.lblod.info/id/bestuurseenheden/8d6609b4-1a20-46b8-97ed-150c6d53acd6> AS ?org)
+  BIND(<http://data.lblod.info/id/persoon/05ab337fad326fa39ecdfc2aa0bad35b> AS ?mockPerson)
+  BIND(<http://data.lblod.info/id/account/8bcc4a2df7d2f45274fdd911342795eb> AS ?mockAccount)
+
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?org ?orgP ?orgO .
+  }
+
+  GRAPH ?g {
+    ?mockPerson ?mockPersonP ?mockPersonO .
```

Comment:
Deleteing mock accounts is not needed. If the bestuurseenheid is deleted, https://github.com/lblod/update-bestuurseenheid-mock-login-service takes care of the rest